packages/1155-contracts/tsconfig.json

```diff
@@ -1,23 +1,7 @@
 {
+  "extends": "tsconfig/tsconfig.json",
```

Comment:
would it be smart to call this "@zoralabs/tsconfig" even it it's private in the case there is a package collision?

Reply:
not a bad idea - also would improve understandability of what the import references.  will do.

Reply:
👍

Reply:
done